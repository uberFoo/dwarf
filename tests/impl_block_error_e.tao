struct Foo {}

impl Foo {
    fn new() -> Self {
        Foo {}
    }

    struct Bar {}

    fn bar(self) -> Self {
        self
    }
}