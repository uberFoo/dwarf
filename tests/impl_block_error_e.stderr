[31mError:[0m impl blocks may only contain functions.
    [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0mimpl_block_error_e:8:5[38;5;246m][0m
    [38;5;246mâ”‚[0m
 [38;5;246m 8 â”‚[0m [31mâ•­[0m[31mâ”€[0m[31mâ–¶[0m[31m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[31ms[0m[31mt[0m[31mr[0m[31mu[0m[31mc[0m[31mt[0m[31m [0m[31mB[0m[31ma[0m[31mr[0m[31m [0m[31m{[0m[31m}[0m
 [38;5;240m   â”†[0m [31mâ”†[0m   
 [38;5;246m10 â”‚[0m [31mâ”œ[0m[31mâ”€[0m[31mâ–¶[0m[31m [0m[31m [0m[31m [0m[31m [0m[31m [0m[31mf[0m[31mn[0m[38;5;249m [0m[38;5;249mb[0m[38;5;249ma[0m[38;5;249mr[0m[38;5;249m([0m[38;5;249ms[0m[38;5;249me[0m[38;5;249ml[0m[38;5;249mf[0m[38;5;249m)[0m[38;5;249m [0m[38;5;249m-[0m[38;5;249m>[0m[38;5;249m [0m[38;5;249mS[0m[38;5;249me[0m[38;5;249ml[0m[38;5;249mf[0m[38;5;249m [0m[38;5;249m{[0m
 [38;5;240m   â”‚[0m [31mâ”‚[0m                                
 [38;5;240m   â”‚[0m [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m [31mimpl blocks may only contain functions.[0m
[38;5;246mâ”€â”€â”€â”€â•¯[0m

